CMakeLists.txt

```diff
@@ -306,7 +306,7 @@ OCV_OPTION(WITH_OPENJPEG "Include JPEG2K support (OpenJPEG)" ON
 OCV_OPTION(WITH_JPEG "Include JPEG support" ON
   VISIBLE_IF TRUE
   VERIFY HAVE_JPEG)
-OCV_OPTION(WITH_JPEGXL "Include JPEG XL support" OFF
```

Comment:
I added `-DWITH_JPEGXL=ON` to Ubuntu 24.04 configuration. No need to hack CMake any more.

Reply:
Thank you for your comment, I reverted it !